app/presenters/content_item/featured_attachments_migration.rb

```diff
@@ -0,0 +1,22 @@
+module ContentItem
+  module FeaturedAttachmentsMigration
+    def choose_field(new_field_name:, old_field_name:)
+      new_list = content_item["details"].fetch(new_field_name, [])
+      old_list = content_item["details"].fetch(old_field_name, [])
+
+      if new_list != []
```

Comment:
You could do `return old_list unless new_list.any?` here, then you don't need to nest the rest of this in the if block.